panc/src/test/java/org/quattor/pan/PanTest.java

```diff
@@ -0,0 +1,95 @@
+package org.quattor.pan;
+
+import org.quattor.pan.exceptions.SyntaxException;
+import java.util.*;
+
+/**
+ * Created by iliclaey.
+ *
+ * When building the panc-project a panc-MAJOR.MINOR-SNAPSHOT-fat-jar-with-tests.jar file is created.
+ * This jar can be used to execute pan test files and should be used as follows:
+ * `java -Dpanc.testdir="path" -Dpanc.tmpdir="path" -jar path/to/panc-MAJOR.MINOR-SNAPSHOT-fat-jar-with-tests.jar`.
```

Comment:
If the tests succeed, `$?` will be 0, otherwise it should be 1. When there's something wrong with the pan code itself or with the expected outcome, the correct error message will be displayed as well.